src/app/public/theming/theme-if.directive.spec.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  SkyThemeModule
+} from '../theme.module';
+
+import {
+  SkyThemeIfTestComponent
+} from './fixtures/theme-if-test.component';
+
+import {
+  SkyThemeSettings
+} from './theme-settings';
+
+import {
+  SkyTheme
+} from './theme';
+
+import {
+  SkyThemeMode
+} from './theme-mode';
+
+import {
+  SkyThemeService
+} from './theme.service';
+
+describe('ThemeIf directive', () => {
+  let fixture: ComponentFixture<SkyThemeIfTestComponent>;
+  const defaultThemeSettings = new SkyThemeSettings(SkyTheme.presets.default, SkyThemeMode.presets.light);
+  const modernThemeSettings = new SkyThemeSettings(SkyTheme.presets.modern, SkyThemeMode.presets.dark);
+  const themeService = new SkyThemeService();
```

Comment:
If you're going to construct `SkyThemeService` yourself (which I recommend not doing in the next comment), `themeService` should be set inside a `beforeEach()` rather than inside the `describe()`. `SkyThemeService` is stateful, and this change will keep tests from leaking state into one another which we've seen can cause failures if tests ever run in a different order. The latest version of Jasmine runs tests in random order, and even though we disable it, a developer could come in and rearrange tests or add a new test in the middle and get unexpected results.